crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -440,6 +440,63 @@ MAP
     ..@...
 ENDMAP
 
+NAME:    mainiacjoe_lair_zoo
+TAGS:    lair_entry transparent
+TAGS:    no_monster_gen no_item_gen
+ORIENT:  float
+SHUFFLE: NALPS
```

Comment:
The cages need to be masked with no_tele_into, since there's no way out if the player gets teleported in and doesn't have another teleport/dig/blink (admittedly rather unlikely, but possible).